src/main/java/com/homework/demo/message/ResponseMessage.java

```diff
@@ -0,0 +1,18 @@
+package com.homework.demo.message;
+
+public class ResponseMessage {
```

Comment:
Try to use Lombok for all repeated code.
Have a look at those annotation: 

- Getter

- Setter

- FieldDefaults

- RequiredArgsConstructor